### ISSUE TITLE
The tool doesn't work on ubuntu out of the box

### ISSUE THREAD

**Issue by schneuwlym** [coddingtonbear/python-myfitnesspal#28]
I just installed your tool and got an error:

```
ERROR:myfitnesspal.cmdline:No recommended backend was available
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/myfitnesspal/cmdline.py", line 46, in main
    COMMANDS[args.command[0]]['function'](args, *extra)
  File "/usr/local/lib/python2.7/dist-packages/myfitnesspal/commands.py", line 79, in store_password
    store_password_in_keyring(args.username, password)
  File "/usr/local/lib/python2.7/dist-packages/myfitnesspal/keyring_utils.py", line 35, in store_password_in_keyring
    password,
  File "/usr/local/lib/python2.7/dist-packages/keyring/core.py", line 48, in set_password
    _keyring_backend.set_password(service_name, username, password)
  File "/usr/local/lib/python2.7/dist-packages/keyring/backends/fail.py", line 18, in get_password
    raise RuntimeError("No recommended backend was available")
RuntimeError: No recommended backend was available
Traceback (most recent call last):
  File "/usr/local/bin/myfitnesspal", line 9, in <module>
    load_entry_point('myfitnesspal==1.8.0', 'console_scripts', 'myfitnesspal')()
  File "/usr/local/lib/python2.7/dist-packages/myfitnesspal/cmdline.py", line 46, in main
    COMMANDS[args.command[0]]['function'](args, *extra)
  File "/usr/local/lib/python2.7/dist-packages/myfitnesspal/commands.py", line 79, in store_password
    store_password_in_keyring(args.username, password)
  File "/usr/local/lib/python2.7/dist-packages/myfitnesspal/keyring_utils.py", line 35, in store_password_in_keyring
    password,
  File "/usr/local/lib/python2.7/dist-packages/keyring/core.py", line 48, in set_password
    _keyring_backend.set_password(service_name, username, password)
  File "/usr/local/lib/python2.7/dist-packages/keyring/backends/fail.py", line 18, in get_password
    raise RuntimeError("No recommended backend was available")
RuntimeError: No recommended backend was available
```

According to another ticket on github, it looks like something has been changed...
https://github.com/zalando/zmon-cli/issues/10

Installing keyrings.alt solved my issue. So I guess this has to be added to the requirements.txt

**Comment by coddingtonbear** [comment]
Awesome; thanks for posting this!

**Comment by coddingtonbear** [comment]
Fixed via 832feaf9fd035611c92e6b42be0572daeb494619 and released as part of 1.8.1.  Cheers!
